printer_mods/doubletrouble/README.md

```diff
@@ -0,0 +1,2 @@
+V0 Skirt mod
+One piece, dual color, built in V0 Display and C14 in the rear
```

Comment:
You have 3 images with your mod, can you include them in this README?

    ![Skirt 1](Skirt_1.jpg)
    ![Skirt 2](Skirt_2.jpg)
    ![Skirt 3](Skirt_3.jpg)

Reply:
Think everything should be fixed now